core/examples/http_bench.rs

```diff
@@ -272,7 +267,13 @@ fn op_close(rid: i32) -> Box<HttpBenchOp> {
   }))
 }
 
-fn op_read(rid: i32, zero_copy_buf: Option<PinnedBuf>) -> Box<HttpBenchOp> {
```

Comment:
Your patch includes some superfluous elements still, like changing the signature of this function.

Like another comment I made, I understand that you're sketching out how this would be in the more complex setting of `cli`, but I would nevertheless prefer that we keep this example as simple as possible. For example there shouldn't be a need for the `is_sync` parameter. All ops are know they are sync or async.

Reply:
Done